### DISCUSSION TITLE
Hidden pages/groups appearing in search

### DISCUSSION THREAD

**Discussion by augustoortolantr** [GitbookIO/community#684] [0. General bugs and questions] [answered]
The hiding/enabling groups and pages functionality was a great addition to the gitbook.
However, when we search for content that contains a hidden page, it appears in the search list.
I believe that hidden content should not appear in the search, only those that have already been released for reading.

![image](https://github.com/user-attachments/assets/5b06752b-4147-489a-b4cc-9861fe8c6aab)

![image](https://github.com/user-attachments/assets/ab131c2b-7925-4cc8-88a0-3e9521850184)

![image](https://github.com/user-attachments/assets/6a690582-5564-446c-8508-a18800ea34bf)

**Comment by addisonschultz** [comment]
Hi @augustoortolan - thanks for the feedback. It's known that the pages will still be indexed by search: https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space#hide-a-page-from-the-table-of-contents

But we'll take your feedback as we continue to iterate on this feature!

**Comment by aikoii** [comment, marked as answer]
Hey @augustoortolan we just shipped this feature 🎉. Check out #730 for more details. Thanks again for your feedback 👋

**Comment by augustoortolantr** [comment]
Thanks!!!
